Sources/ArcGISToolkit/Components/Popups/PopupView.swift

```diff
@@ -77,27 +80,29 @@ public struct PopupView: View {
     
     public var body: some View {
         VStack(alignment: .leading) {
-            HStack {
-                if !popup.title.isEmpty {
-                    Text(popup.title)
-                        .font(.title)
-                        .fontWeight(.bold)
-                }
-                Spacer()
-                if showCloseButton {
```

Comment:
By removing `showCloseButton` from here now, users can avoid a deprecation warning and begin the new practice of controlling the close button's visibility via the binding in the initializer.